packages/cli/commands/customObject.js

```diff
@@ -18,22 +20,24 @@ exports.builder = yargs => {
 
   console.warn(
     chalk.reset.yellow(
-      '***************************** WARNING WARNING WARNING ****************************'
-    )
-  );
-  console.warn(
-    chalk.reset.yellow(
-      'The Custom Object CLI is currently in beta and is subject to change.'
+      `***************************** ${i18n(`${i18nKey}.warning`)} ${i18n(
+        `${i18nKey}.warning`
+      )} ${i18n(`${i18nKey}.warning`)} ****************************`
     )
   );
+  console.warn(chalk.reset.yellow(i18n(`${i18nKey}.betaMessage`)));
```

Comment:
Not a blocker, but I do wonder if we should register a handlebars helper for color or yellow or something?

Reply:
Good idea. I added this, updated the usages of `chalk.reset.yellow` to using `{{#yellow}}<content>{{/yellow}}` in the `en.lyaml` and moved the custom helpers into their own js file.